pkg/cmd/projectCreate.go

```diff
@@ -660,6 +709,48 @@ func createTestFolderGradle(fileGenerator *utils.FileGenerator, targetTestFolder
 	return err
 }
 
+// Creates a build.gradle and a bnd.bnd file in a Gradle test project directory.
+func createManagerFolder(fileGenerator *utils.FileGenerator, targetTestFolderPath string,
+	packageName string, featureName string, forceOverwrite bool, isDevelopment bool) error {
+
+	type TestGradleParameters struct {
+		Parent      GradleCoordinates
+		Coordinates GradleCoordinates
+		// Version of Galasa we are targetting
+		GalasaVersion string
+		IsDevelopment bool
+	}
+
+	galasaVersion, err := embedded.GetGalasaVersion()
+
+	if err == nil {
+		gradleProjectTemplateParameters := TestGradleParameters{
+			Parent:        GradleCoordinates{GroupId: packageName, Name: packageName},
+			Coordinates:   GradleCoordinates{GroupId: packageName, Name: packageName + "." + featureName},
+			GalasaVersion: galasaVersion,
+			IsDevelopment: isDevelopment}
+
+		buildGradleFile := utils.GeneratedFileDef{
+			FileType:                 "gradle",
+			TargetFilePath:           targetTestFolderPath + "/build.gradle",
+			EmbeddedTemplateFilePath: "templates/projectCreate/parent-project/test-project/build.gradle.template",
```

Comment:
You will almost certainly need to have your own build.gradle.template for building your generated manager. Suggest you create a new template in `templates/projectCreate/manager-project/build.gradle.template` in the `pkg/embedded/templates/projectCreate/parent-project` folder ?